lib/fluent/plugin/in_kubernetes_metrics_aggregator.rb

```diff
@@ -66,29 +66,35 @@ def get_cpu_mult(cpu)
         end
 
         # https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory
+        # 1 Ki = 1024 bytes
+        # 1 K = 1000 bytes = 1000/1024 Ki = 1000/1024*1024 Mi
         def get_memory_mult(memory)
-          memory_mult = if memory[-2] == 'Ki'
-                          0.001
-                        elsif memory[-2] == 'K'
-                          1.0 / 1024
-                        elsif memory[-2] == 'Mi'
+          memory_mult = if memory[-2..] == 'Ki'
```

Comment:
[-2] will return 2nd last character from the string and [-2..] will return last 2 characters from string.
For example, in case of '1000Ki', [-2] will return 'K' and [-2..]  will return 'Ki'.